akvo/rsr/models/project.py

```diff
@@ -1593,6 +1593,64 @@ def update_parents(self, update_period, difference, sign):
         except (InvalidOperation, TypeError):
             pass
 
+    @classmethod
+    def log_project_addition(cls, project_id, user):
+        project = cls.objects.get(id=project_id)
+        message = u'%s.' % (_(u'Project editor, added project'))
+
+        LogEntry.objects.log_action(
+            user_id=user.pk,
+            content_type_id=ContentType.objects.get_for_model(project).pk,
+            object_id=project.pk,
+            object_repr=project.__unicode__(),
+            action_flag=ADDITION,
+            change_message=message
+        )
+
+        # Perform IATI checks after a project has been created.
+        project.update_iati_checks()
+
+    @staticmethod
+    def add_custom_fields(project_id, organisations):
+        from akvo.rsr.models import OrganisationCustomField, ProjectCustomField
+        custom_fields = OrganisationCustomField.objects.filter(
+            organisation__in=organisations
+        )
+        copy_fields = (
+            'name', 'type', 'section', 'order', 'max_characters', 'mandatory', 'help_text'
+        )
+        project_custom_fields = [
+            ProjectCustomField(
+                project_id=project_id,
+                **{field: getattr(custom_field, field) for field in copy_fields}
+            )
+            for custom_field in custom_fields
+        ]
+        ProjectCustomField.objects.bulk_create(project_custom_fields)
+
+    @classmethod
+    def new_project_created(cls, project_id, user):
+        """Hook to do some book-keeping for a newly created project.
+
+        *NOTE*: This hook cannot be moved into a post-save hook since we need
+        information about the user who created this project, to perform some of
+        the actions.
+
+        """
+        # Set reporting organisation
+        organisation_ids = user.approved_employments().values_list('organisation_id', flat=True)
```

Comment:
I think this should be limited to orgs where we know the user can create projects.

This is of course part of the larger problem of the org being selected essentially randomly, but let's not deal with that for now.

Reply:
Updated the PR to set only orgs where the user can create a project as the reporting org. Nice catch!